src/main/java/ch/heigvd/pro/b04/votes/ServerVote.java

```diff
@@ -25,6 +26,22 @@
   @Getter
   private boolean answerChecked;
 
+  @Override
+  public int compareTo(ServerVote o) {
+    if (idVote.getTimeVote().after(o.getIdVote().getTimeVote())) {
+      return -1;
+    } else if (idVote.getTimeVote().before(o.getIdVote().getTimeVote())) {
+      return 1;
+    } else {
+      return 0;
+    }
+  }
```

Comment:
Same as above.

Reply:
Database  cannot sort by itself, because we have no way to write manually that request

Reply:
https://www.baeldung.com/spring-data-sorting